edge-util/src/Microsoft.Azure.Devices.Edge.Util/VerifyTwinSignature.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Cryptography;
+    using System.Security.Cryptography.X509Certificates;
+
+    using Org.Webpki.JsonCanonicalizer;
+    public class VerifyTwinSignature
```

Comment:
Is this twin specific? If not, can we call it SignatureValidator or something liek that?

Reply:
Thats a good point. It can be re-used for any json data